src/docs/sphinx/quickstart_guide/config_build.rst

```diff
@@ -276,76 +307,151 @@ directly to CMake; for example::
+OpenMP, CUDA (NVIDIA GPUs), and HIP (AMD GPUs), are implemented using RAJA. See
+:ref:`dependencies-label` for instructions to set the directory location of a
+RAJA installation. In addition, enabling such features may require additional build 
+options to be provided, which are summarized in the following table.
+
++------------------------------+------------------------------------------+
+| OPTION                       | Description                              |
++==============================+==========================================+
+| RAJA_DIR                     | RAJA installation directory as described |
+|                              | in :ref:`dependencies-label`.            |
+|                              |                                          |
+|                              | RAJA must be built with support enabled  |
+|                              | for OpenMP, CUDA, or HIP to use features |
+|                              | enabled by those parallel programming    |
+|                              | models in Axom.                          |
++------------------------------+------------------------------------------+
+| CUDA_TOOLKIT_ROOT_DIR        | Path to CUDA software stack installation |
++------------------------------+------------------------------------------+
+| CMAKE_CUDA_COMPILER          | Path to CUDA compiler (e.g., nvcc)       |
++------------------------------+------------------------------------------+
+| CMAKE_CUDA_ARCHITECTURES     | Target architecture(s) for CUDA          |
++------------------------------+------------------------------------------+
+| HIP_ROOT_DIR                 | Path to HIP software stack installation  |
++------------------------------+------------------------------------------+
+| CMAKE_HIP_ARCHITECTURES      | Target architecture(s) for HIP           |
++------------------------------+------------------------------------------+
+
+.. important:: Depending on the version of RAJA you are using, you may need to
+               set the C++ standard to C++14. RAJA requires C++14 as of release
+               v2022.03.0.
 
 .. note :: To configure the version of the C++ standard, you can supply one of the
            following values for **BLT_CXX_STD**:  'c++11' or 'c++14'.
            Axom requires at least 'c++11', the  default value.
 
-See :ref:`dependencies-label` for configuration variables to specify paths
-to Axom's dependencies.
+
+Tools and features primarily intended for developers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++------------------------------------------+---------+----------------------------------------+
+| OPTION                                   | Default | Description                            |
++==========================================+=========+========================================+
+| AXOM_ENABLE_TOOLS                        | ON      | Enable Axom development tools          |
```

Comment:
As I noted above, I think the `AXOM_ENABLE_TOOLS` option should be moved back to the first table in this section.

Reply:
done.